main/zsh/APKBUILD

```diff
@@ -2,15 +2,16 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=zsh
-pkgver=5.4.1
+pkgver=5.4.2
 pkgrel=0
 pkgdesc="A very advanced and programmable command interpreter (shell)"
 url="http://www.zsh.org/"
 arch="all"
 license="BSD"
+options="!check" # As of 5.4.2 - 3 tests fails
 makedepends="ncurses-dev"
 install="zsh.post-install zsh.post-upgrade zsh.pre-deinstall"
-source="http://www.zsh.org/pub/$pkgname-$pkgver.tar.xz
+source="http://www.zsh.org/pub/$pkgname-$pkgver.tar.gz
```

Comment:
strange that `.xz` is missing upstream..

Reply:
I thought the same. Guess it is released by hand and somebody forgot.

Reply:
I emailed the authors. One minor improvement for this abuild could be switching to https which seems to be working just fine.

Reply:
Ok. Which email address did you use? I need it to report the failing tests.

Reply:
At the [main site](http://zsh.sourceforge.net) there's a maintainers link in the footer. Not sure if that's most accurate, but I thought it was worth a shot.

Reply:
Sourceforge seems to have the `xz` tarball; don't know if its worth switching.